main.go

```diff
@@ -244,7 +244,7 @@ func Run(ctx context.Context) int {
 	}
 	// TODO: remove as soon as we update the configuration with statsExcludedTags where necessary
```

Comment:
Can someone explain this TODO to me?

Reply:
Was discussing the same with @chandumlg ... So the basic idea is that for free customers we don't want to send so many tags to influx as it gets bloated up but essentially we should remove the redundant tags for every metric